docs/TranslationGuide.md

```diff
@@ -235,3 +243,4 @@ const testSuite = [
 [src/assets/app.config.sample.json]:                       ../src/assets/app.config.sample.json
 [Configuration.md]:                                        ./Configuration.md
 [zonemaster.conf-example]:                                 ../zonemaster.conf-example
+[Poedit]:                                                  [https://poedit.net]
```

Comment:
typo: no need for brackets around the URL

Reply:
Or rather, the brackets should not be there. As it is now the link does not work.

Reply:
Oops, fixed